tests/test_signal.py

```diff
@@ -371,3 +371,16 @@ def test_flatten():
 
     npt.assert_allclose(signal_in._data.reshape((6, -1)), signal_out._data)
     assert id(signal_in) != id(signal_out)
+
+
+def test___eq___equal(signal):
+    sine = np.sin(2 * np.pi * 440 * np.arange(0, 1, 1 / 44100))
+    actual = Signal(sine, 44100, len(sine), domain='time')
```

Comment:
This would fail, if something in conftest changes, maybe the approach with signal.copy() would be better?